bl.c

```diff
@@ -555,6 +545,7 @@ cout_word(uint32_t val)
 	cout((uint8_t *)&val, 4);
 }
 
+#ifndef DISABLE_FOR_IO
```

Comment:
I think we want to garbage collect. If we need to disable weith #define use a feature like NO_CIN_WORD and not the HW name, That way HW can share it as a feature.